docs/installation_python.md

```diff
@@ -0,0 +1,229 @@
+# Python Installation
+
+!!! note
+      This version of the documentation is for installation of Xailient SDK using wheel file (.whl). 
+      If you are looking for documentation for installion of Xailient SDK using SDK package, please visit our old [documentation](https://xailient.readthedocs.io/en/latest/installation/).
+      
+      If your local machines doesn't meet the requirements for native installation we also offer Docker [Containers as a deployment option](https://xailient-docs.readthedocs.io/en/latest/container/).
+
+
+## Device Setup
+
+To meet the minimum requirements, your device must be a x86_64 or ARM32 architecture device. We've tested the following operating systems:
+
+* Ubuntu 18.04 LTS
+* Ubuntu 19.04
+* Raspbian Buster
+
+!!! note
+
+    To run Xailient SDK on Windows or MacOS, please run it inside a VirtualBox.
+
+    Here are some resources you can refer to to install Ubuntu operating system on a VirtualBox:
+
+    * [Resource 1](https://brb.nci.nih.gov/seqtools/installUbuntu.html)
+    * [Resource 2](https://medium.com/@tushar0618/install-ubuntu-16-04-lts-on-virtual-box-desktop-version-30dc6f1958d0)
+    * [Resource 3](https://itsfoss.com/install-linux-in-virtualbox/)
+
+
+For ARM32, we have tested on the following device:
+
+* RaspberryPi 3B+
+
+__Minimum software requirements:__
+
+* Glibc 2.27
+* Python >=3.6
+* Pip
+* VirtualEnv
+* libcurl4 
+* php-curl
+
+If your device does not meet the minimum software required, please follow the instructions below to install them.
+
+
+### Installing PYTHON SDK
+
+!!! note
+
+    If you are installing C++ SDK, please refer to this [section](https://xailient-docs.readthedocs.io/en/latest/container/).
+
+__Download the package lists and update information__
+
+```bash
+$ sudo apt-get update
+```
+
+!!! note
+    apt-get update to download the package lists from the repositories and "update" them to get information on the newest versions of packages and their dependencies. It will do this for all repositories and PPAs.
+
+__Install python3__
+
+The Xailient SDK works with python 3.6 or later. Here we install python 3.7 though you could install any version
+of python >=3.6.
+
+```bash
+$ sudo apt-get install python3.7
+```
+
+__Install pip for python3__
+
+```bash
+$ sudo apt-get install python3-pip
+```
+
+__Install other required libraries__
+
+```bash
+$ sudo apt-get install libcurl4 php-curl
```

Comment:
Do we really need these still? I'm not 100% sure we need them anymore but it's been a while since I've had to install the C++ SDK so I could be wrong.

Reply:
This one is for the Python SDK. Most of the devices have these libraries pre-installed. However, when Sanjid was going through the installation, we found the package missing, which caused some issue during installation. Hence, included in the installation step.